apiserver/facades/client/cloud/cloud_test.go

```diff
@@ -1312,17 +2097,35 @@ func (s *cloudSuite) TestModifyCloudUpdateAccess(c *gc.C) {
 		},
 	})
 	c.Assert(err, jc.ErrorIsNil)
-	s.backend.CheckCallNames(c, "Cloud", "ControllerTag", "CreateCloudAccess", "GetCloudAccess", "UpdateCloudAccess")
-	s.backend.CheckCall(c, 2, "CreateCloudAccess", "fluffy", names.NewUserTag("fred"), permission.AdminAccess)
-	s.backend.CheckCall(c, 4, "UpdateCloudAccess", "fluffy", names.NewUserTag("fred"), permission.AdminAccess)
 	c.Assert(results.Results, gc.DeepEquals, []params.ErrorResult{
 		{},
 	})
 }
 
-func (s *cloudSuite) TestModifyCloudAlreadyHasAccess(c *gc.C) {
-	s.backend.cloudAccess = permission.AdminAccess
-	results, err := s.api.ModifyCloudAccess(params.ModifyCloudAccessRequest{
+func (s *cloudSuiteGomocks) TestModifyCloudAlreadyHasAccess(c *gc.C) {
+	bruceTag := names.NewUserTag("bruce")
+	defer s.setup(c, bruceTag).Finish()
```

Comment:
I think we should be admin for this test

Reply:
Yes for sure, think I forgot to change it after testing.